package/cifs/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:16.04
+RUN apt-get update && \
+    apt-get install -y jq curl nfs-common
```

Comment:
I used ubuntu vanilla 16:04 as my VM os & did not install any packages in os level.
Maybe `nfs-common` is already preinstalled in ubuntu 16.04.
lets add both of these packages in Dockerfile, agreed?